salt/cloud/clouds/vmware.py

```diff
@@ -3031,9 +3031,6 @@ def create_snapshot(name, kwargs=None, call=None):
             '-a or --action.'
         )
 
-    if kwargs is None:
-        kwargs = {}
-
     snapshot_name = kwargs.get('snapshot_name') if kwargs and 'snapshot_name' in kwargs else None
```

Comment:
@techhat If ``kwargs`` is None, then ``snapshot_name`` will not be equal to ``kwargs.get('snapshot_name')`` and instead will be equal to ``None``. How is this going to fail?